### DISCUSSION TITLE
Do we need to build the CLI every time for the "docs" workflow?

### DISCUSSION THREAD

**Discussion by pat-s** [woodpecker-ci/woodpecker#2201] [General]
See https://ci.woodpecker-ci.org/repos/3780/pipeline/7259/3.

I think we could save time and resources by just downloading the latest CLI binary?

**Comment by qwerty287** [comment]
It must run using the current way. The docs generation uses [a custom entry point](https://github.com/woodpecker-ci/woodpecker/blob/main/cmd/cli/docs.go) so we can't just download a binary.
Also downloading a binary does not respect the changes done after the release.

**Comment by qwerty287** [comment]
We can't improve this currently as [explained above](https://github.com/woodpecker-ci/woodpecker/discussions/2201#discussioncomment-6707904), I'm closing it for now.
